docs/src/main/asciidoc/security-csrf-prevention.adoc

```diff
@@ -9,11 +9,25 @@ include::_attributes.adoc[]
 
 https://owasp.org/www-community/attacks/csrf[Cross-Site Request Forgery (CSRF)] is an attack that forces an end user to execute unwanted actions on a web application in which they are currently authenticated.
 
-Quarkus Security provides a CSRF prevention feature which implements a https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html#double-submit-cookie[Double Submit Cookie technique]. This techninque requires that the CSRF token is never directly exposed to scripts executed on the client-side. In this extension, the CSRF token is:
+Quarkus Security provides a CSRF prevention feature which implements https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html#double-submit-cookie[Double Submit Cookie] and [CSRF Request Header] techniques.
 
-* sent as `HTTPOnly` cookie to the client, and
+`Double Submit Cookie` techninque requires that the CSRF token is never directly exposed to scripts executed on the client-side. CSRF token is:
```

Comment:
I'm not sure what it means "never directly exposed to scripts executed on the client-side" because if it's in a form element, then scripts can access it. What matters is that it's not exposed to scripts with a different origin than the endpoint.

Reply:
@FroMage Sure, that needs to be clarified, the CSRF cookie that is HttpOnly